src/main/java/com/autotune/analyzer/workerimpl/BulkJobManager.java

```diff
@@ -0,0 +1,229 @@
+        allExperiments.add(experimentName);
+        return allExperiments;
+    }
+
+    @Override
+    public void run() {
+        try {
+
+            String uniqueKey = null;
+            // Process labels in the 'include' section
+            if (this.bulkInput.getFilter() != null && this.bulkInput.getFilter().getInclude() != null) {
+                // Initialize StringBuilder for uniqueKey
+                StringBuilder includeLabelsBuilder = new StringBuilder();
+                Map<String, String> includeLabels = this.bulkInput.getFilter().getInclude().getLabels();
+                if (includeLabels != null && !includeLabels.isEmpty()) {
+                    includeLabels.forEach((key, value) ->
+                            includeLabelsBuilder.append(key).append("=").append("\"" + value + "\"").append(",")
+                    );
+                    // Remove trailing comma
+                    if (includeLabelsBuilder.length() > 0) {
+                        includeLabelsBuilder.setLength(includeLabelsBuilder.length() - 1);
+                    }
+                    LOGGER.info("Include Labels: " + includeLabelsBuilder.toString());
+                    uniqueKey = includeLabelsBuilder.toString();
+                }
+            }
+            DataSourceMetadataInfo metadataInfo = null;
+            DataSourceManager dataSourceManager = new DataSourceManager();
+            DataSourceInfo datasource = CommonUtils.getDataSourceInfo("prometheus-1");
+
+
+            if (null != this.bulkInput.getTime_range() && this.bulkInput.getTime_range().getStart() != null && this.bulkInput.getTime_range().getEnd() != null) {
+                // Extract interval start and end times
+                String intervalEndTimeStr = this.bulkInput.getTime_range().getStart();
+                String intervalStartTimeStr = this.bulkInput.getTime_range().getEnd();
+                long interval_end_time_epoc = 0;
+                long interval_start_time_epoc = 0;
+                LocalDateTime localDateTime = LocalDateTime.parse(intervalEndTimeStr, DateTimeFormatter.ofPattern(KruizeConstants.DateFormats.STANDARD_JSON_DATE_FORMAT));
+                interval_end_time_epoc = localDateTime.toEpochSecond(ZoneOffset.UTC);
+                Timestamp interval_end_time = Timestamp.from(localDateTime.toInstant(ZoneOffset.UTC));
+                localDateTime = LocalDateTime.parse(intervalStartTimeStr, DateTimeFormatter.ofPattern(KruizeConstants.DateFormats.STANDARD_JSON_DATE_FORMAT));
+                interval_start_time_epoc = localDateTime.toEpochSecond(ZoneOffset.UTC);
+                Timestamp interval_start_time = Timestamp.from(localDateTime.toInstant(ZoneOffset.UTC));
+
+                int steps = 15 * KruizeConstants.TimeConv.NO_OF_SECONDS_PER_MINUTE;
+                //Get metaData
+                //example metadataInfo = dataSourceManager.importMetadataFromDataSource(datasource, uniqueKey, interval_start_time_epoc, interval_end_time_epoc, steps);
```

Comment:
Looks like redundant code, please remove them

Reply:
The commented code will be uncommented in the next PR; it was only used as a placeholder for now